auto-pr-script/create_pull_requests

```diff
@@ -21,8 +21,8 @@ branches=("$@")
 
 function main(){
 	# Use the author info of the original Git commit
-	git config --global user.email $(git show -s --format='%ae' $commit_sha)
-	git config --global user.name $(git show -s --format='%an' $commit_sha)
+	git config --global user.email "$(git show -s --format='%ae' "$commit_sha")"
+	git config --global user.name "$(git show -s --format='%an' "$commit_sha")"
```

Comment:
The command `git show -s --format='%an' "$commit_sha"` extracts the commit author but if it contains non standard char, the `git config` command could fail.